libs/components/lookup/src/lib/modules/lookup/types/lookup-show-more-native-picker-async-context.ts

```diff
@@ -7,20 +7,20 @@ import { SkyLookupShowMoreNativePickerConfig } from './lookup-show-more-native-p
  * @internal
  * Context for the show more native picker. These values are provided by the lookup component.
  */
-export interface SkyLookupShowMoreNativePickerAsyncContext {
-  descriptorProperty: string;
+export class SkyLookupShowMoreNativePickerAsyncContext {
```

Comment:
Since all of these are required, it probably makes sense to make all of these constructor parameters so an instance can't be created without setting those properties. Same goes for the other classes with the non-null assertions on their properties.

```
constructor(public descriptorProperty: string, public idProperty: string, ...) {}
```